starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -61,9 +63,44 @@ public void create() {
     inventory.setProductType(PRODUCT_TYPE);
     inventory.setId(ID);
     Inventory actualInventoryItem = this.inventoryDAO.create(inventory);
+    
     Assert.assertNotNull(actualInventoryItem);
-    Assert.assertEquals(inventory.getProductType(), actualInventoryItem.getProductType());
     Assert.assertEquals(inventory.getName(), actualInventoryItem.getName());
+    Assert.assertEquals(inventory.getProductType(), actualInventoryItem.getProductType());
     Assert.assertNotEquals(actualInventoryItem.getId(), ID);
   }
+  
+  /**
+   * Test Delete method where the inventory to be deleted exists
+   */
+  @Test
+  public void deleteInventoryExists() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    this.inventoryDAO.create(inventory);
+    String createdID = inventory.getId();
+    Optional<Inventory> deletedInventory = this.inventoryDAO.delete(createdID);
```

Comment:
`createdId` can be replaced with the getter here; remove the `createdId` variable afterwards.

Reply:
Changes made in commit f0f86e97102500321c70556b975a146356f674fd